app/views/curation_concerns/base/_iiif_fields.html.erb

```diff
@@ -27,5 +27,9 @@
         </div>
       <% end %>
     </div>
+
+    <div>
+      <%= f.input :ocr_language, collection: [:eng, :ita], input_html: { name: "#{f.object.model_name.singular}[ocr_language][]", multiple: true, class: 'resource-radio-button' } %>
```

Comment:
Contemplating shelling out so tesseract can list available languages. I'll give it a shot.